WooCommerce/Classes/ViewModels/Order Details/OrderDetailsViewModel.swift

```diff
@@ -82,6 +83,10 @@ final class OrderDetailsViewModel {
                                       cardPresentPaymentsConfiguration: configurationLoader.configuration)
     }()
 
+    private(set) lazy var editNoteViewModel: EditCustomerNoteViewModel = {
+        return EditCustomerNoteViewModel(order: order)
+    }()
```

Comment:
I decided to define as lazy because there is no point in being alive until the first time the merchant wants to update the note provided by the customer. WDYT? 🤔